README.md

```diff
@@ -104,7 +104,7 @@ The built-in Options renderer also support custom classNames, just add a `classN
 
 You can enable multi-value selection by setting `multi={true}`. In this mode:
 
-* Selected options will be removed from the dropdown menu
+* Selected options will be removed from the dropdown menu by default. If you want them to remain in the list, set `removeSelected={true}`
```

Comment:
@banderson  just a minor nit: should this  be "If you want them to remain in the list, set `removeSelected={false}`"?